### PR TITLE
[Feature] Add initial Storybook support

### PR DESCRIPTION
**Pull request by seriouslysean**
## Please check the following

- [x] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle#run-the-tests))
- [x] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle#linting-formatting))
- For New Features:
  - [x] Have tests been added to cover any new features or fixes?
  - [x] Has the documentation been updated accordingly?

## Please describe additional details for testing this change
- Pull down this branch
- Run `npm ci`
- Run `npm run storybook`
- Rejoice 👏🏼 

image

### TODO
- Add Vuetify to Storybook (maybe?)
    - https://github.com/nidkil/vuetify-with-storybook
    - https://www.c-od-e.com/2019/06/30/vuejs-vuetify-and-storybook-integration/
    - https://github.com/desiboli/storybook-vue-vuetify
- Style global storybook theme
- Test github pages deployment
- Add/Update documentation
- Identify components that would make sense in Storybook

### CODE REVIEW

**Review by peterancourt — APPROVED** [review]
Nice looks good 🎉

**Review by itsalaidbacklife — COMMENTED** [review]
Looks good. Gonna run locally then approve

File: README.md [hunk 1]
```diff
@@ -103,6 +109,13 @@ npm run lint
 ```
 to check the formatting without autofixing problems. (This is what's run in CI when a PR is opened against the `main` branch).
 
+#### Storybook (UX Documentation)
+Run Storybook locally with
+```
+npm run storybook
+```
+ It will start automatically on localhost:6006 and open a new tab in your browser.
```
Comment (itsalaidbacklife) [inline README.md:117]: Do we need to run `npm run build-storybook` before we can run this?

**Review by itsalaidbacklife — APPROVED** [review]
Looks great, works nicely

File: README.md (hunk 1 above)
Comment (itsalaidbacklife) [reply README.md:117]: I'm fuzzy on what build is for. Just for deploying to gh-pages?

**Pull request merged**